### DISCUSSION TITLE
Move between tabs

### DISCUSSION THREAD

**Discussion by parzival418** [mathuo/dockview#343] [Q&A]
I am looking to implement a similar behaviour where a user can use keyboard shortcuts to move between tabs, similar to the behaviour in chrome using command+alt+arrow.  How would I go about programmatically switching between different tabs?

Thanks!

**Comment by mathuo** [comment]
There is support to switch the active panel progmatically in the dockview api.

It could probably do with a few upgrades such as a flag to move between groups or between tabs in a group (current it just toggles all tabs in one group and then moves onto the next group) and a rollover option where if it reaches the last tab it rolls back to the first tab but let me add an example and improve the documentation around these methods as they are not well documented.

Take a look at this example, is this along the lines of what you are looking for? https://codesandbox.io/s/keyboard-dockview-fkwm5h

**Reply by parzival418** [reply]
This is perfect! Thank you!
